proxy/deploy_test.go

```diff
@@ -14,43 +14,42 @@ import (
 	"github.com/openfaas/faas-cli/test"
 )
 
-func Test_DeployFunction(t *testing.T) {
-	s := test.MockHttpServerStatus(
-		t,
-		http.StatusOK, // DeleteFunction
-		http.StatusOK, // DeployFunction
-	)
-	defer s.Close()
-
-	stdout := test.CaptureStdout(func() {
-		DeployFunction(
-			"fproces",
-			s.URL,
-			"function",
-			"image",
-			"language",
-			true,
-			nil,
-			"network",
-			[]string{},
-			false,
-			[]string{},
-			map[string]string{},
-			FunctionResourceRequest{},
-		)
-	})
+type DeployProxyTest struct {
+	title               string
+	mockServerResponses []int
+	replace             bool
+	update              bool
+	expectedOutput      string
+}
 
-	r := regexp.MustCompile(`(?m:Deployed.)`)
-	if !r.MatchString(stdout) {
-		t.Fatalf("Output not matched: %s", stdout)
-	}
+var DeployProxyTests = []DeployProxyTest{
```

Comment:
The capital letter means this is exported outside the package. Was that the intention?

Reply:
Was not the intention, I'll fix this.